bundle/Resources/public/js/views/plugins/cof-copycontentplugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-copycontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the Copy Content plugin.
+     *
+     * @module cof-copycontentplugin
+     */
+    Y.namespace('cof.Plugin');
+
+    /**
+     * Provides plugin for the copy content plugin.
+     *
+     * @namespace cof.Plugin
+     * @class CopyContent
+     * @constructor
+     * @extends eZ.Plugin.CopyContent
+     */
+    Y.cof.Plugin.CopyContent = Y.Base.create('cofCopyContentPlugin', Y.eZ.Plugin.CopyContent, [], {
```

Comment:
Using `isSelectable` function was my first idea how to solve this problem, but I realize in `contentStruct` there is passing: `location`, `content`, `contentType` and `contentInfo`. In this step I cannot pass all of those models. Based on that I don't know what can be needed in `isSelectable` function so I drop this idea.
I could have all this information after publishing a content, so the user would have to create whole content and then he would go back to choose content type and do everything from the start, it's bad UX I think.